### PR TITLE
secure logistic regression inference based on HE and SGX

### PR DESCRIPTION
**Pull request by xhuan28**

### CODE REVIEW

File: cczoo/lr_infer_he_sgx/src/infer_client.cpp [hunk 1]
```diff
@@ -0,0 +1,170 @@
+
```
Comment (RodgerZhu) [inline cczoo/lr_infer_he_sgx/src/infer_client.cpp:14]: Please add :
# Copyright (C) 2022 Intel Corporation
# SPDX-License-Identifier: Apache-2.0

File: cczoo/lr_infer_he_sgx/src/infer_client.hpp [hunk 2]
```diff
@@ -0,0 +1,51 @@
+#ifndef INFER_CLIENT_HPP_
```
Comment (RodgerZhu) [inline cczoo/lr_infer_he_sgx/src/infer_client.hpp:15]: Please add:
# Copyright (C) 2022 Intel Corporation
# SPDX-License-Identifier: Apache-2.0

File: cczoo/lr_infer_he_sgx/src/infer_server.cpp [hunk 3]
```diff
@@ -0,0 +1,196 @@
+#include <algorithm>
```
Comment (RodgerZhu) [inline cczoo/lr_infer_he_sgx/src/infer_server.cpp:15]: Please add:
# Copyright (C) 2022 Intel Corporation
# SPDX-License-Identifier: Apache-2.0

File: cczoo/lr_infer_he_sgx/src/infer_server.hpp [hunk 4]
```diff
@@ -0,0 +1,41 @@
+#ifndef INFER_SERVER_HPP_
```
Comment (RodgerZhu) [inline cczoo/lr_infer_he_sgx/src/infer_server.hpp:15]: Please add:
# Copyright (C) 2022 Intel Corporation
# SPDX-License-Identifier: Apache-2.0

File: cczoo/lr_infer_he_sgx/src/param.cpp [hunk 5]
```diff
@@ -0,0 +1,41 @@
+#include "param.hpp"
```
Comment (RodgerZhu) [inline cczoo/lr_infer_he_sgx/src/param.cpp:15]: Please add:
# Copyright (C) 2022 Intel Corporation
# SPDX-License-Identifier: Apache-2.0

File: cczoo/lr_infer_he_sgx/src/utils.cpp [hunk 6]
```diff
@@ -0,0 +1,88 @@
+#include "utils.hpp"
```
Comment (RodgerZhu) [inline cczoo/lr_infer_he_sgx/src/utils.cpp:21]: Please add:
# Copyright (C) 2022 Intel Corporation
# SPDX-License-Identifier: Apache-2.0

File: cczoo/lr_infer_he_sgx/src/utils.hpp [hunk 7]
```diff
@@ -0,0 +1,21 @@
+#ifndef UTILS_HPP_
```
Comment (RodgerZhu) [inline cczoo/lr_infer_he_sgx/src/utils.hpp:15]: Please add:
# Copyright (C) 2022 Intel Corporation
# SPDX-License-Identifier: Apache-2.0

File: cczoo/lr_infer_he_sgx/test/unit_test.cpp [hunk 8]
```diff
@@ -0,0 +1,10 @@
+#include <gtest/gtest.h>
```
Comment (RodgerZhu) [inline cczoo/lr_infer_he_sgx/test/unit_test.cpp]: Please add:
# Copyright (C) 2022 Intel Corporation
# SPDX-License-Identifier: Apache-2.0

File: cczoo/lr_infer_he_sgx/cmake/gflags.cmake [hunk 9]
```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2020 Intel Corporation
```
Comment (RodgerZhu) [inline cczoo/lr_infer_he_sgx/cmake/gflags.cmake:1]: 2020->2022

File: cczoo/lr_infer_he_sgx/src/infer_client.hpp (hunk 2 above)
Comment (xhuan28) [reply cczoo/lr_infer_he_sgx/src/infer_client.hpp:15]: resolved.

**Pull request merged**